scalability_and_performance/index.adoc

```diff
@@ -0,0 +1,63 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="index"]
+= {product-title} Scalability and Performance overview
+include::_attributes/common-attributes.adoc[]
+:context: index
+
+// ifndef::openshift-dedicated,openshift-rosa[]
+{product-title} offers best practices and utilities to help you optimize the performance and scale of your {product-title} clusters. This book in the documentation provides information on recommended performance and scalability practices, reference design specifications, optimization and low latency tuning.
+
+To get support on these topics, refer to the section of the documentation entitled xref:../support/getting-support.adoc#getting-support[Getting support]
+
+// endif::openshift-dedicated,openshift-rosa[]
+
+[NOTE]
+====
+Operators described in this section have release cycles that are separate from the core {product-title} release cycles. The release schedule for the Node Observability Operator can be found here link:access.redhat.com/support/policy/updates/openshift_operators[Openshift Operators]
+====
+
+The Scalability and Performance section of the documentation offers the following topics:
+
+[discrete]
+==== Recommended performance and scalability practices
+
+xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.adoc#recommended-control-plane-practices[Recommended control plane practices]
+
+xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.adoc#recommended-infrastructure-practices[Recommended infrastructure practices]
+
+xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#recommended-etcd-practices[Recommended etcd practices]
+
+[discrete]
+==== Reference design specifications
```

Comment:
Optional: You could be more specific here, since the links are all Telco RAN: 

==== Telco Reference design specifications

Up to you.

Reply:
Good point. I was trying to sound general but my examples are from Telco, so I'll be honest about that.